docs/updating.md

```diff
@@ -70,17 +84,17 @@ repos:
           # Prevent committing .rej files
           - id: forbidden-files
             name: forbidden files
-            entry: found Copier update rejection files; review them and remove them
+            entry: found Copier update rejection files; review and remove them before merging.
```

Comment:
This was actually incorrect before: [`entry`](https://pre-commit.com/#hooks-entry) refers to the executable to run. This is a [description](https://pre-commit.com/#hooks-description):

```suggestion
            description: found Copier update rejection files; review and remove them before merging.
```